main.go

```diff
@@ -588,7 +588,7 @@ func printList() {
 	}
 	b.WriteString("\n\nOther Events:              Sets:\n")
 	b.WriteString("____________               ____\n\n")
-	for i := int(tracee.SysEnterEventID); i < len(tracee.EventsIDToEvent); i++ {
+	for i := int(tracee.SysEnterEventID); i < int(tracee.MaxEventID); i++ {
```

Comment:
is this change needed for the fix?

Reply:
The length of the map is now smaller than the maximum event id.
There is a gap between the last syscall number (arch dependent) and common events (starting from 1000)